src/v/config/configuration.cc

```diff
@@ -2437,9 +2437,10 @@ configuration::configuration()
       "memory occupied by objects that can be carried over from one trim to "
       "next, and allows cache to quickly unblock readers before starting the "
       "directory inspection.",
-      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      {.needs_restart = needs_restart::no,
+       .visibility = visibility::deprecated},
       // This roughly translates to around 1000 carryover file names
```

Comment:
the comment has to be updated

Reply:
updated

Reply:
I mean the comment that says "This roughly translates to around 1000 carryover file names"

Reply:
if we set the visibility to deprecated I don't see a reason to also include it in the description.